### ISSUE TITLE
Someone please add Llama 3.2 11b to the leaderboard 

### ISSUE THREAD

**Issue by droidXrobot** [open-compass/VLMEvalKit#493]

**Comment by shan23chen** [comment]
Does the repo support this model yet? Thanks!

**Comment by FangXinyu-0913** [comment]
Hi @droidXrobot @shan23chen! This repo now supports Llama-3.2-11B/90B-Vision-Instruct, you can use it with the newest transformers version (>=4.45.0.dev0)!
However, the evaluation results obtained based on the current repo do not match the official results, and after the hyperparameters and the system prompt are aligned, there is still the problem of more dropped accuracy (mainly for ai2d). Is there anyone willing to solve this problem?

Ref:
https://huggingface.co/meta-llama/Llama-3.2-11B-Vision-Instruct/blob/main/generation_config.json
https://github.com/meta-llama/llama-models/blob/main/models/llama3_2/eval_details.md

**Comment by luohao123** [comment]
Actually, all my benchmark can not align same model as before...

THIS REPO UPDATE TOO QUICK... Many things might casued not alignment..

**Comment by kennymckormick** [comment]
> Actually, all my benchmark can not align same model as before...
> 
> THIS REPO UPDATE TOO QUICK... Many things might casued not alignment..

Would you please provide more information, such as the corresponding commit ID of the previous and current code you used for evaluation, as well as the model & benchmark you have evaluated?

**Comment by luohao123** [comment]
As for user, we can not compare each commit to see what changed, it's your responsibility.

The current situation, all benchmark are drop almost can treat wrong evaluation, The changes I can observe is:

1. The tsv file new generated;
2. This operation not have before, I donkt now what is this:
![image](https://github.com/user-attachments/assets/9a857a1e-2f5d-4444-a06e-adcdd5b5b66f)
and it is slow
3. the metric now all lower, on all benchmarks, same model

I even doubt is my training codebase got wrong, stuck me about 1 week,

afterwards, I relise, the evaluation pipeline was broken, the old model can not repeat the metric before.

Any suggestion?

**Comment by kennymckormick** [comment]
> As for user, we can not compare each commit to see what changed, it's your responsibility.
> 
> The current situation, all benchmark are drop almost can treat wrong evaluation, The changes I can observe is:
> 
> 1. The tsv file new generated;
> 2. This operation not have before, I donkt now what is this:
>    ![image](https://private-user-images.githubusercontent.com/49749220/374845397-9a857a1e-2f5d-4444-a06e-adcdd5b5b66f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0NTkzNDUsIm5iZiI6MTcyODQ1OTA0NSwicGF0aCI6Ii80OTc0OTIyMC8zNzQ4NDUzOTctOWE4NTdhMWUtMmY1ZC00NDQ0LWEwNmUtYWRjZGQ1YjViNjZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA5VDA3MzA0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTExZjkyZTYxYTliMTdhNGU3MjU4YjUyNzFjOTE2Y2RiODc5ODc0M2M4YzQ0ODNjNzFjMzgzM2M5MDA1YzAyZTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.i6blHsToVcP_3nQ_M60Xk5JG5Wd_Jl9ESh6yEqhcpIM)
>    and it is slow
> 3. the metric now all lower, on all benchmarks, same model
> 4. I dont know what changed inside the evalkit.
> 
> I even doubt is my training codebase got wrong, stuck me about 1 week,
> 
> afterwards, I relise, the evaluation pipeline was broken, the old model can not repeat the metric before.
> 
> Any suggestion?

At least, you need to provide some information so that we can help. Please tell me the model you are using, one / several benchmarks you are evaluating. If you cannot find out the initial commit you are using, please try to remember when you first use this codebase.

**Comment by kennymckormick** [comment]
> As for user, we can not compare each commit to see what changed, it's your responsibility.
> 
> The current situation, all benchmark are drop almost can treat wrong evaluation, The changes I can observe is:
> 
> 1. The tsv file new generated;
> 2. This operation not have before, I donkt now what is this:
>    ![image](https://private-user-images.githubusercontent.com/49749220/374845397-9a857a1e-2f5d-4444-a06e-adcdd5b5b66f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NDE2MDYsIm5iZiI6MTcyODU0MTMwNiwicGF0aCI6Ii80OTc0OTIyMC8zNzQ4NDUzOTctOWE4NTdhMWUtMmY1ZC00NDQ0LWEwNmUtYWRjZGQ1YjViNjZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEwVDA2MjE0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlMDc0YWQ0ZDUyY2E1YjdiZDQwZjczMzNkZjBjNmQ1ZWI0YjAxNDhjMzhlYWVlZGJjOWFjMmY2YjQ5Y2E2ZTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.o2EI64cNcYOXAne4D5DoyFR0a8BFgOU3PRYO9KWOldk)
>    and it is slow
> 3. the metric now all lower, on all benchmarks, same model
> 4. I dont know what changed inside the evalkit.
> 
> I even doubt is my training codebase got wrong, stuck me about 1 week,
> 
> afterwards, I relise, the evaluation pipeline was broken, the old model can not repeat the metric before.
> 
> Any suggestion?

Same Here: https://github.com/open-compass/VLMEvalKit/issues/503#issuecomment-2404134873

Also, if you want to go further with this problem, maybe creating a new issue is a better idea. You problem is not related to the issue of llama-3.2.

**Comment by terry-for-github** [comment]
> > As for user, we can not compare each commit to see what changed, it's your responsibility.
> > The current situation, all benchmark are drop almost can treat wrong evaluation, The changes I can observe is:
> > 
> > 1. The tsv file new generated;
> > 2. This operation not have before, I donkt now what is this:
> >    ![image](https://private-user-images.githubusercontent.com/49749220/374845397-9a857a1e-2f5d-4444-a06e-adcdd5b5b66f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NDE2MDYsIm5iZiI6MTcyODU0MTMwNiwicGF0aCI6Ii80OTc0OTIyMC8zNzQ4NDUzOTctOWE4NTdhMWUtMmY1ZC00NDQ0LWEwNmUtYWRjZGQ1YjViNjZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEwVDA2MjE0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlMDc0YWQ0ZDUyY2E1YjdiZDQwZjczMzNkZjBjNmQ1ZWI0YjAxNDhjMzhlYWVlZGJjOWFjMmY2YjQ5Y2E2ZTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.o2EI64cNcYOXAne4D5DoyFR0a8BFgOU3PRYO9KWOldk)
> >    and it is slow
> > 3. the metric now all lower, on all benchmarks, same model
> > 4. I dont know what changed inside the evalkit.
> > 
> > I even doubt is my training codebase got wrong, stuck me about 1 week,
> > afterwards, I relise, the evaluation pipeline was broken, the old model can not repeat the metric before.
> > Any suggestion?
> 
> Same Here: [#503 (comment)](https://github.com/open-compass/VLMEvalKit/issues/503#issuecomment-2404134873)
> 
> Also, if you want to go further with this problem, maybe creating a new issue is a better idea. You problem is not related to the issue of llama-3.2.

@kennymckormick Same issue here too. #523 Could you please check this one? Thanks!
